hawkular-wildfly-agent/src/test/java/org/hawkular/agent/monitor/protocol/platform/OshiPlatformCacheTest.java

```diff
@@ -56,51 +56,37 @@ public void testConcurrency() {
         final CountDownLatch doneLatch1 = new CountDownLatch(1);
         final CountDownLatch doneLatch2 = new CountDownLatch(1);
 
-        Thread t1 = new Thread(new Runnable() {
```

Comment:
this is a test - no biggie. I would just leave everything as is. The other changes were minimal. I'm ok with leaving this PR as is if its working and passing tests.